config/database.yml

```diff
@@ -0,0 +1,84 @@
+# PostgreSQL. Versions 9.3 and up are supported.
+#
+# Install the pg driver:
+#   gem install pg
+# On macOS with Homebrew:
+#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
+# On Windows:
+#   gem install pg
+#       Choose the win32 build.
+#       Install PostgreSQL and put its /bin directory on your path.
+#
+# Configure Using Gemfile
+# gem "pg"
+#
+default: &default
+  adapter: postgresql
+  encoding: "LATIN1"
```

Comment:
- [ ] I was unable to set up your project since the encoding you are using is does not match the local project type. This `"LATIN1"` should be `utf-8` to fix this error.

  ![image](https://github.com/agneta1991/social-media-app/assets/36542986/197ef5ab-0ea2-47eb-87d5-157e0d69d50b)

Reply:
on my end if i have this set to utf-8 i am not able to do anything with postgress as i have errors exact same ones like you pasted in the picture, could it be because i am using Linux ubuntu??
Could that be the reason? i have spent almost 6 hours trying to figure it out and failed to find the solution for it. any advise what should I do? i could change it manually before submitting it every time to make sure this does not happen, but maybe you could help me figuring it out to avoid these sort of errors? @TedLivist

Reply:
Hmm @agneta1991 that's interesting. I used Linux Ubuntu too. Perhaps you can leave a comment for the next reviewer.

Reply:
for the time being i just changed it to the one you have written in the comment, and later when i work on my end i will change it back. and play like this until i figure out the underlyig issue. thank you so much for your response! @TedLivist